iocore/net/QUICNet.cc

```diff
@@ -73,10 +73,12 @@ QUICPollCont::_process_long_header_packet(QUICPollEvent *e, NetHandler *nh)
   ptype = static_cast<QUICPacketType>(buf[0] & 0x7f);
   switch (ptype) {
   case QUICPacketType::INITIAL:
-    vc->read.triggered = 1;
-    vc->handle_received_packet(p);
-    this->mutex->thread_holding->schedule_imm(vc, QUIC_EVENT_PACKET_READ_READY);
-    return;
+    if (!vc->read.triggered) {
+      vc->read.triggered = 1;
+      vc->handle_received_packet(p);
+      vc->handleEvent(QUIC_EVENT_PACKET_READ_READY, nullptr);
```

Comment:
Using direct call instead of schedule event since the qvc may be closed in the period of event creating  and processing.

Currently only QVC::acceptEvent will set `read.enabled = 1`, then all packet could be processed as normal. **So this change may not be necessary **.